starter/scenes.js

```diff
@@ -0,0 +1,194 @@
+            if (confirmFight) {
+                var useHelmet = confirm("You see a helmet! Wanna use it?\n");
+                if (useHelmet) {
+                    userPoints -= 10;
+                    alert("Hemingway trashes you:\n\"A real man does not use a helmet, but nevermind, let's fight!\"");
+                } else if (!useHelmet) {
+                    userPoints += 10;
+                    alert("Hemingway praises you:\n\"A real man fights, and in the real man's way!\"");
+                }
+                var resultOfFight = fightHemingway(useHelmet);
+                if (resultOfFight === "won") {
+                    alert("Hemingway says:\n\"You're the real man!\"\nHe then introduced you to Picasso.");
+                } else if (resultOfFight === "draw") {
+                    alert("Hemingway says:\n\"A real man must fight, and you did!\"\nHe then buys you a drink.");
+                } else if (resultOfFight === "lost") {
+                    alert("Hemingway says:\n\"At least you fought honourably!\"\nHe made you buy him a drink.");
+                }
+            } else {
+                alert("Hemingway calls you a chicken!");
+                userPoints -= 5;
+            }
+        } else if (poetType === "B") {
+            userPoints += 30;
+            alert("He's totally not weird! Because he is Tom Hiddleston.... ahem, I meant Fitzgerald!");
+        } else {
+            alert("You don't know how to play a game, eh?");
+        }
+    } else {
+        alert("You don't know how to play a game, eh?");
+    }
+}
+
+var future = function() {
+    var futureType = prompt("You think that the unforeseen will be the golden age? Why?\n I can make it happen! (A)\n Aliens might rescue us! (B)");
+    if (futureType === "A") {
+        userPoints += 10;
+        var confidentType = prompt("You really think you could change the future?\nFor better (A) or for worse (B)?");
+        if (confidentType === "A") {
+            userPoints += 10;
+            alert("Confident! I hope you see to it though!");
+        } else if (confidentType === "B") {
+            userPoints += 20;
+            alert("Ok, all hail to the king / queen!");
+        } else {
+            alert("You don't know how to play a game, eh?");
+        }
+    } else if (futureType === "B") {
+        userPoints += 20;
+        var alienExist = prompt("Do you believe that aliens are among us?\nYes (Y) or No (N)");
+        if (alienExist === "Y") {
+            userPoints += 20;
+            alert("Woot! I believe it too!\nMaybe it's just too many episodes of Stargates?");
+        } else if (alienExist === "N") {
+            userPoints -= 5;
+            alert("You are ridiculous, look around you more often!");
+        } else {
+            alert("You don't know how to play a game, eh?");
+        }
+    } else {
+        alert("You don't know how to play a game, eh?");
```

Comment:
If you find yourself writing something that's repeated multiple times, it'll be good to either put it in a function. or declare it in a variable like:

var taunt = "You don't know how to play a game, eh?"
alert(taunt);